test_apis.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests each output API against the output API schema"""
+
+import jsonschema
+import os
+import pytest
+from json import load
+
+
+API_ENDING = "_output_API.json"
+SCHEMA_FILE_NAME = "output_API.schema.json"
+
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+SCHEMA_PATH = os.path.join(BASE_DIR, SCHEMA_FILE_NAME)
+
+with open(SCHEMA_PATH, "r") as f:
+    try:
+        schema = load(f)
+    except Exception as e:
+        raise Exception(f"Failed loading json {SCHEMA_PATH}") from e
+
+
+def test_schema():
+    """Ensure the output_API.schema.json conforms to JSON Schema Draft 7"""
+    jsonschema.Draft7Validator.check_schema(schema)
```

Comment:
i think this happens on line 33 anyways - when you create a validator on a schema, it checks it. Please check if it is. maybe then we don't need this `test_schema`

Reply:
`Draft7Validator` inherits from `IValidator`:
https://python-jsonschema.readthedocs.io/en/stable/validate/#jsonschema.IValidator

Specifically says that it assumes passed schema is valid, and that you should run `check_schema` first

Reply:
Otherwise we could do use `jsonschema.validate()` and either implicitly use the metaschema from `$schema` or explicitly pass `Draft7Validator`.
`jsonschema.validate()` checks that the schema is valid first, while `IValidator.validate()` does not.

Reply:
It would then also revalidate the schema itself every time it checks an `...output_API.json` file which increases overhead a little